### PR TITLE
add pending button to skip campus event if published

### PR DESCRIPTION
**Pull request by bingzhang**
## Description
_Please provide a summary of the pull request and the issue it fixes. Please add necessary details, context, dependencies, explanation of when review is needed (see next section), etc._

**Fixes #(add issue number here and remove parentheses)**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [ ] Within a week
- [x] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [x] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [x] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

File: templates/events/calendar.html [hunk 1]
```diff
@@ -58,20 +58,20 @@ <h1 class="text-center">{{title}}
...
-      <div class="form-check form-check-inline">
-          <input type="checkbox" class="form-check-input" id="materialInline2" {% if "disapproved" in select_status %}checked{%endif%}>
-          <label class="form-check-label" for="materialInline2">Disapproved</label>
+        <label class="form-check-label" for="materialInline1">Pending</label>
       </div>
+{#      <div class="form-check form-check-inline">#}
+{#          <input type="checkbox" class="form-check-input" id="materialInline2" {% if "disapproved" in select_status %}checked{%endif%}>#}
+{#          <label class="form-check-label" for="materialInline2">Disapproved</label>#}
+{#      </div>#}
       <div class="form-check form-check-inline">
```
Comment (sandeep-ps) [inline templates/events/calendar.html:67]: I think the "published" checkbox should be selected by default. Thanks.

File: templates/events/event.html [hunk 2]
```diff
@@ -57,6 +61,29 @@ <h5 class="modal-title" id="exampleModalLabel">
...
+      <div class="modal-header">
+        <h5 class="modal-title" id="exampleModalLabel">
+          Delete event
+        </h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        You are about to pending the following event <b>{{post.title}}</b>
```
Comment (sandeep-ps) [inline templates/events/event.html:78]:
```suggestion
        You are about to set the following event <b>{{post.title}}</b> to pending state.
```

File: templates/events/calendar.html [hunk 3]
```diff
@@ -58,20 +58,20 @@ <h1 class="text-center">{{title}}
       <div class="form-check form-check-inline"><h4>Show</h4></div>
       <div class="form-check form-check-inline">
         <input type="checkbox" class="form-check-input" id="materialInline1" {% if "approved" in select_status %}checked{%endif%}>
-        <label class="form-check-label" for="materialInline1">Approved</label>
-      </div>
-      <div class="form-check form-check-inline">
-          <input type="checkbox" class="form-check-input" id="materialInline2" {% if "disapproved" in select_status %}checked{%endif%}>
-          <label class="form-check-label" for="materialInline2">Disapproved</label>
+        <label class="form-check-label" for="materialInline1">Pending</label>
```
Comment (sandeep-ps) [inline templates/events/calendar.html:61]: Let's use Bootstrap Pill Badges like User events. 
https://github.com/rokwire/events-manager/blob/skip_campus_events_GUI/templates/events/user-events.html#L39
https://getbootstrap.com/docs/4.0/components/badge/#pill-badges. 
Let's use badge-success for Published and badge-warning for Pending. Thanks.

File: templates/events/event.html [hunk 4]
```diff
@@ -92,7 +119,7 @@ <h5>Event Status </h5>
               <p class="col-sm-9">
                 {% if post.eventStatus == "approved" %}
                     <!-- temporily change to published -->
-                  <span class="badge badge-pill badge-success"> published </span>
+                  <span class="badge badge-pill badge-success"> pending </span>
```
Comment (sandeep-ps) [inline templates/events/event.html:122]: Let's use warning class to match the pending style.
```suggestion
                  <span class="badge badge-pill badge-warning"> pending </span>
```

**Comment by sandeep-ps** [comment]
@bingzhang, the "Publish" button seems to be missing from user events. Please check if this is caused due to changes in this PR. Thanks.

**Comment by sandeep-ps** [comment]
A pending calendar event needs to have a "Publish" button too. Thanks.

File: templates/events/calendar.html (hunk 1 above)
Comment (bingzhang) [reply templates/events/calendar.html]: i double checked the user event, it seems user event does not have the published checked by default.

Comment (sandeep-ps) [reply templates/events/calendar.html]: This has been fixed by Bing.

**Review by sandeep-ps — APPROVED** [review]

**Pull request merged**